projects/canopy/src/lib/carousel/auto-play/auto-play.component.scss

```diff
@@ -12,5 +14,9 @@
     height: var(--space-xxl);
     font-size: var(--text-fs-3-size);
     cursor: pointer;
+
+    &:focus {
+      @include lg-outer-focus-outline(var(--default-focus-color));
```

Comment:
This is ok but just FYI you don't have to pass the default colour since it's applied automtically.

Reply:
Yes, I was expecting it to apply the default colour but but for some reason it was not and it came as an accessibility bug in testing so added the style.